src/app/components/PasswordForm/index.tsx

```diff
@@ -86,13 +93,21 @@ export default function PasswordForm({
     <>
       <div className="w-full mb-6">
         <TextField
-          autoFocus
+          autoFocus={autoFocus}
           id="password"
           label={t("choose_password.label")}
           type={passwordView ? "text" : "password"}
           required
           onChange={handleChange}
           tabIndex={0}
+          minLength={minLength}
```

Comment:
This could also be done in a follow-up PR, right? Just if this leads to further discussions/effort.

Reply:
Do you mean the TODO to add translation for the min length validation, or something else?

Reply:
This whole validation part. Just in case if this is sparking any dicussion. If not, even better to have this.